homeassistant/components/tuya/strings.json

```diff
@@ -187,6 +187,18 @@
           "4": "Mood 4",
           "5": "Mood 5"
         }
+      },
+      "weather_delay": {
+        "state": {
+          "cancel": "Cancel",
+          "1day": "24h",
+          "2days": "48h",
+          "3days": "72h",
+          "4days": "96h",
+          "5days": "120h",
+          "6days": "144h",
+          "7days": "168h"
```

Comment:
Why change the format from the native values?

Reply:
I basically set it in the same way as it's shown in the SmartApp UI from Tuya.
I also thing it's more using friendly then having to calculate hours to days.

Reply:
> I basically set it in the same way as it's shown in the SmartApp UI from Tuya.

That is not correct to do. The UI from Tuya can be customized for each manufacturer for each device type they have (which are over hundreds of thousands of devices).

Please use the specification provided instead.

Reply:
Fixed